benchmarks/mwt/setup/01-cassandra-install/01-install-crd.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - command: kubectl kudo install cassandra --skip-instance --namespace cassandra  
```

Comment:
we should test from the local path in this case if MWT scripts are living next to operator

Reply:
@zmalik I don't know what you mean

Reply:
if you are referring to using the local build of this operator in this repo... I would rather not... MWT should only be on released operators and the scripts should work against the released operator repo IMO

Reply:
I would generally agree, although with a project that is heavily in development this might not always be possible. For the last MWT I used a dev-version from master.

I do hope that we have reached a state where this won't be necessary anymore though.

Reply:
we usually participate with the next unreleased version in MWT to make sure there are no stability issues with the upcoming release under load tests.
that's a non-blocker issue as would require minimal effort to change it when running MWT. Moreover, now given the new dates we will participate with a released version so each time it will heavily dependent on operator release dates and MWT dates